.travis.yml

```diff
@@ -1,4 +1,8 @@
+os:
+- linux
 language: node_js
+node_js:
```

Comment:
See if we can remove these changes and see if it still works?

Reply:
sure thing. updated. works as expected.